src/meta/meta_bulk_load_service.cpp

```diff
@@ -1651,9 +1657,28 @@ void bulk_load_service::try_to_continue_app_bulk_load(
                                different_status_pidx_set));
 }
 
+// ThreadPool: THREAD_POOL_META_SERVER
+/*static*/ bool
+bulk_load_service::validate_ingest_behind(const std::map<std::string, std::string> &envs, bool val)
+{
+    bool app_allow_ingest_behind = false;
+    const auto &iter = envs.find(replica_envs::ROCKSDB_ALLOW_INGEST_BEHIND);
+    if (iter != envs.end()) {
+        if (!buf2bool(iter->second, app_allow_ingest_behind)) {
+            dwarn_f("can not convert {} to bool", iter->second);
+            app_allow_ingest_behind = false;
+        }
+    }
+    if (val && !app_allow_ingest_behind) {
```

Comment:
`return app_allow_ingest_behind && val;`
And I think it's better to rename value to ingest_behind

Reply:
The return value could not simplify as `app_allow_ingest_behind && val`, the return value is like:
val && app_allow_ingest_behind -> true
!val && !app_allow_ingest_behind -> true
!val && app_allow_ingest_behind -> true
val && !app_allow_ingest_behind -> false
By the way, rename val is a good idea, I have updated it.